pod/perllocale.pod

```diff
@@ -696,12 +722,12 @@ or if you have the "env" application you can do (in any shell)
 If you do not know what shell you have, consult your local
 helpdesk or the equivalent.
 
-=head2 Permanently fixing locale problems
+=head3 Permanently fixing locale problems
 
 The slower but superior fixes are when you may be able to yourself
 fix the misconfiguration of your own environment variables.  The
 mis(sing)configuration of the whole system's locales usually requires
-the help of your friendly system administrator.
+the help of your (hopefully) friendly system administrator.
```

Comment:
Please consider deleting all of `(hopefully) friendly`.

Reply:
done